genesis/utils/array_class.py

```diff
@@ -2044,11 +2045,37 @@ class StaticRigidSimCacheKey:
     integrator: int = cache_value()
     sparse_solve: bool = cache_value()
     solver_type: int = cache_value()
+    # dynamic properties
```

Comment:
what do you mean by 'dynamic'? Do you mean, you intend to move this from static config into ndarray/fields later? Maybe add `# TODO`, and a short descriptin?